test/unit/decorators.test.ts

```diff
@@ -157,6 +158,39 @@ describe("Decorators", () => {
     )
   })
 
+  describe("@Link", () => {
+    let BaseModel: typeof SpraypaintBase
+
+    beforeEach(() => {
+      @Model()
+      class MyBase extends SpraypaintBase {}
+      BaseModel = MyBase
+    })
+
+    it("adds to the link list", () => {
+      @Model()
+      class TestClass extends BaseModel {
+        @Link() link1!: string
+      }
+
+      expect(TestClass.linkList).to.include("link1")
+    })
+
+    it("child class adds to the link list of parent", () => {
+      @Model()
+      class TestClass extends BaseModel {
+        @Link() link1!: string
+      }
+
+      class ChildTestClass extends TestClass {
+        @Link() link2!: string
+      }
+
+      expect(ChildTestClass.linkList).to.include("link1")
+      expect(ChildTestClass.linkList).to.include("link2")
```

Comment:
I'm almost positive it doesn't, but would add an assertion that `link2` is not in the parent list.

Reply:
Sure, agree with that.